src/enochecker/results.py

```diff
@@ -67,10 +68,21 @@ def __init__(
         self,
         *args: Any,
         scoreboard_message: Optional[str] = None,
+        internal_message: Optional[str] = None,
         **kwargs: Dict[Any, Any]
     ):
-        super().__init__(*args)
-        self.message = scoreboard_message
+        super().__init__(*args, **kwargs)
```

Comment:
`kwargs` needs to be removed here

Reply:
No, why?

Reply:
`Exception` only expects `args` , from the cli:
`src/enochecker/results.py:74: error: Too many arguments for "__init__" of "BaseException"`